### PR TITLE
Status - System Logs - Manage Log

### PR DESCRIPTION
**Pull request by NOYB**
Build up th manage log section with options to override the "General Logging Options" settings on an individual log basis.
Remove over exuberant gettext's.
Set/adjust filter form field widths to be better fitting for the field types.
Open/Close filter form based on filtering state.

Doesn't restart syslog daemon for changes it is not required for.
 Forward/Reverse Display
 GUI Log Entires
 Formated/Raw Display
 Web Server Log (enable/disable)

### CODE REVIEW

File: src/usr/local/www/diag_logs.php [hunk 1]
```diff
@@ -377,20 +486,141 @@ function getGETPOSTsettingvalue($settingname, $default) {
...
+
+$group->add(new Form_Checkbox(
+	'format',
+	null,
+	'General Logging Options Setting',
+	($pconfig['format'] == '') ? true : false,
+	''
+))->displayAsRadio();
+
+$group->setHelp('Show the log entries as formated or raw output as generated by the service. The raw output will reveal more detailed information, but it is more difficult to read.');
```
Comment (phil-davis) [inline src/usr/local/www/diag_logs.php]: s/formated/formatted/

**Comment by heper** [comment]
@NOYB just out of curiosity: why'd you remove the gettext ? doesn't this remove the possibility for translations?

**Comment by sbeaver-netgate** [comment]
gettext and htmlspecialchars are applied where required within the Form_* framework so are no longer required when composing the page. See Section.class.php line 77 for an example.

Steve
--
Stephen Beaver | +1 (512) 646-4100;239 | sbeaver@netgate.com | www.netgate.com

**Comment by heper** [comment]
thanks, good to know.

**Comment by sbeaver-netgate** [comment]
Would it be worth making the table of formatted logs sortable? I can imagine that clicking a column header to sort by process or by PID might be useful/ It would only require changing the table declaration to:
````html
<table class="table table-striped table-hover table-condensed sortable-theme-bootstrap" data-sortable>
````

**Comment by NOYB** [comment]
Don't know why but making the table sortable messes up the column header vertical alignment.
PID is like it's on it's own line above the others.
snap1

**Comment by sbeaver-netgate** [comment]
I found that the sorting library is very picky about the table code. It must include the full thead/tbody format and the number of columns in the header must exactly match the number of columns in the body. You might check those things.

````html
<table class="table table-striped table-hover table-condensed sortable-theme-bootstrap" data-sortable>
    <thead>
        <<th>One</th>
        <<th>Two</th>
    </thead>
    <tbody>
        <td>One</td>
        <td>Two</td>
    </tbody>
</table>
````

**Pull request closed without merging**